bolts/data/datasets/utils.py

```diff
@@ -19,6 +20,7 @@
     np_str_obj_array_pattern,
     string_classes,
 )
+import torchaudio.transforms as TF_audio
```

Comment:
It should just be T here (the F in the import below stands for 'functional'). That said, T_audio is still quite cumbersome for an alias; maybe AT and we can change the image version to IT or maybe just T as I don't think there will be any collisions.

Reply:
I've gone with just T since the image import is for the functional module as you said.